compiler/src/iree/compiler/Codegen/Dialect/test/micro_kernel_ops.mlir

```diff
@@ -0,0 +1,217 @@
+// RUN: iree-opt --split-input-file --verify-diagnostics %s | FileCheck %s
```

Comment:
I prefer moving invalid op tests to `invalid.mlir`. What do you think?

Reply:
Its confusing to me to split that way. I'd rather keep op roundtrip related tests in one file....

Reply:
ha, sounds like we have different preference. I'm open with both. :D